src/Illuminate/Queue/Worker.php

```diff
@@ -5,14 +5,17 @@
 use Exception;
 use Throwable;
 use Illuminate\Contracts\Events\Dispatcher;
-use Illuminate\Database\DetectsLostConnections;
 use Illuminate\Contracts\Debug\ExceptionHandler;
 use Symfony\Component\Debug\Exception\FatalThrowableError;
 use Illuminate\Contracts\Cache\Repository as CacheContract;
 
 class Worker
 {
-    use DetectsLostConnections;
+    use \Illuminate\Database\DetectsLostConnections, DetectsLostConnections {
+        \Illuminate\Database\DetectsLostConnections::causedByLostConnection insteadof DetectsLostConnections;
+        \Illuminate\Database\DetectsLostConnections::causedByLostConnection as causedByLostDbConnection;
```

Comment:
I think it would be better to add an use statement for this with an alias instead of calling it everytime, would make it a little bit cleaner.